packages/acs-ui-common/src/conditional-compilation-sample/index.tsx

```diff
@@ -175,6 +175,35 @@ function propTrampoline(): string {
  *
  */
 
+/**
+ * Conditional compilation directives can refer to one or more predefined features.
+ *
+ * When working on a beta feature, define a new feature tag in common/config/babel/.babelrc.js
+ * Then use that tag in all the conditional compilation directives for your feature.
+ *
+ * Most examples in this file use the `demo` feature.
+ *
+ * When your feature is ready for stable builds: Move the feature flag to the `stabilizedFeatures` list.
+ * The tagged code will no longer be removed from stable builds.
+ *
+ * Using an undeclared feature in conditional compilation directive will cause a build failure (prevents typos).
+ *
+ * The following code is not removed because it contains a stabilized feature tag.
+ */
+/* @conditional-compile-remove(stabilizedDemo) */
+export const thisIsNowStable = 42;
+
+/* But the following is removed because it contains one non-stabilized feature still. */
```

Comment:
This is not right.

If any feature is stabilized, the associated code should be included in stable build.

If a field is needed by multiple features, the first one that is stabilized needs it to be in the stable build.

Reply:
Done.